packages/bits/spec/components/repeat/repeat.e2e.ts

```diff
@@ -45,68 +39,63 @@ describe("USERCONTROL Repeat", () => {
         expect(await singleSelectList.isCompact()).toEqual(true);
     });
 
-    it("should not be 'striped'", async () => {
-        expect(await multiSelectList.isStriped()).toEqual(false);
-    });
-
-    it("should have the correct header", async () => {
-        expect(await multiSelectList.getHeaderText()).toBe("repeat header from template");
-    });
-
-    it("should apply the correct template", async () => {
-        const blueColorItem = multiSelectList.getItem(0);
-        expect(await blueColorItem.getText()).toEqual("blue");
-    });
-
     it("should allow multi selection of items", async () => {
         await multiSelectList.selectCheckbox(2);
         await multiSelectList.selectCheckbox(5);
-        expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[]");
+        expect(await colorSelection.getText()).toBe("[]");
         await multiSelectList.selectCheckbox(0);
-        expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[ { \"color\": \"blue\" } ]");
-    });
+        expect(await colorSelection.getText()).toBe("[ { \"color\": \"blue\" } ]");
 
-    it("should not allow selection change for disabled items", async () => {
-        const multiSelectListDisabledItems = Atom.find(RepeatAtom, "nui-demo-multi-repeat-disabled-items");
-        expect(await multiSelectListDisabledItems.getCheckbox(0).isDisabled()).toBe(true);
-        expect(await multiSelectListDisabledItems.getCheckbox(4).isDisabled()).toBe(true);
+        // Return to initial state
+        await multiSelectList.selectCheckbox(2);
+        await multiSelectList.selectCheckbox(5);
+        await multiSelectList.selectCheckbox(0);
+        expect(await colorSelection.getText()).toBe("[ { \"color\": \"yellow\" }, { \"color\": \"black\" } ]");
+        await colorSelection.click();
```

Comment:
What does the click on this line do?

Reply:
I didn't find out how to do "SHIFT + TAB" (opposite to Tab in keyboard) such way that it wouldn't select header text either (as `await Helpers.pressKey(Key.SHIFT + Key.TAB);` do). So I just click on page to go out from checkbox. Not sure it's a good idea though.